scriptoocloud/pom.xml

```diff
@@ -19,14 +19,18 @@
 		<admin.password>$2a$04$oQo44vqcDIFRoYKiAXoNheurzkwX9dcNmowvTX/hsWuBMwijqn44i</admin.password>
 
 		<db.driver>com.mysql.jdbc.Driver</db.driver>
-		<db.database.url>jdbc-url-to-db</db.database.url>
+		<db.database.url>jdbc:mysql://localhost:3306/STCDB?useSSL=false&amp;allowPublicKeyRetrieval=true</db.database.url>
```

Comment:
let's not change this

Reply:
Fix reflected in STC-11 and merged into STC-10